tests/test_prepare.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from docpipeline import prepare
+import shutil
+import filecmp
+import yaml
+import pytest
+import pathlib
+import tempfile
+
+
+@pytest.fixture
+def tmpdir_setup(tmpdir):
```

Comment:
Perhaps name this `testdata_dir` or just `testdata`? Kind of awkward naming with the `tmp_path = pathlib.Path(tmpdir_setup)` line below.